testdata/tls/ca.conf

```diff
@@ -0,0 +1,16 @@
+[req]
```

Comment:
please use the already existing `tests/data` rather than `testdata`

Reply:
We already use `testdata` in https://github.com/profianinc/steward and https://github.com/profianinc/drawbridge. It would really want to have these all in one consistent location.

I'll move current files to `tests/data` and file an issue about it

Reply:
Well, since we are nesting these in different subcrates, maybe it's fine as is